Source/WebCore/platform/VideoPixelFormat.cpp

```diff
@@ -26,6 +26,10 @@
 #include "config.h"
 #include "VideoPixelFormat.h"
 
+#if USE(GSTREAMER)
+#include <gst/video/video-format.h>
```

Comment:
afaik, the supported include is `gst/video/video.h`

Reply:
video-format.h is public too, and has smaller impact :)